src/Libraries/DSOffice/DSOffice.csproj

```diff
@@ -10,14 +10,18 @@
     <AssemblyName>DSOffice</AssemblyName>
     <DocumentationFile>$(OutputPath)\$(UICulture)\DSOffice.xml</DocumentationFile>
   </PropertyGroup>
-    <ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
         <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">
             <ExcludeAssets>runtime</ExcludeAssets>
         </PackageReference>
+        <Reference Include="WindowsBase" />
     </ItemGroup>
-  <ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+        <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001" />
```

Comment:
The net48 packageReference is just for compile time. The runtime is excluded because net48 has access to the GAC (That is where we expect the excel interop assembly to be found)
Net6 does not have access to the GAC anymore, so we have to deliver the interop assembly. 
That is why I have 2 different cases for the same package.

Reply:
but they both are the same exact line:
`<PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">`